Add Optimize video settings to Preferences screen

daniloercoli opened the pull request:
This PR adds video transcoding settings to the Preferences screen. Just below the Image setting items.

Video optimization is OFF by default, you need to manually turn it on. (Like to image optimization).
Also note that it's only available for alpha releases, and debug builds.

Minor:
 - Add license info for Mobile 4 Media library

nbradbury commented:
@daniloercoli This all looks good, but I think we could use a description of what "bitrate" is. I know it's in the content description, but not everyone knows to long press to see that. Could we add something very short to this dialog (like "Higher number = higher quality")?

![screenshot_1498060743](https://user-images.githubusercontent.com/3903757/27395059-7c3b02bc-567c-11e7-8bc9-1416dcaa372c.png)

daniloercoli commented:
Good call @nbradbury - I've changed the title of the item to `Video Quality`, to be in par with the *Image Optimize* option, and also changed the selection accordingly. No more bitrate settings exposed to end-users.

nbradbury commented:
"Video Quality" is much more user-friendly! :shipit:
